packages/suite-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Analytics Changelog
 
-This changelog lists all events that Suite tracks.
+This changelog lists changes in suite events.
```

Comment:
Just curious, is CHANGELOG useful here? Since this package is not published and I also think version in package.json do not match any version here anyway :D

Reply:
In the past, it was a source of information for data analysts. It is true, that now we have that information "duplicated" in notion. @matejkriz

Reply:
Notion is internal, I believe it still could be useful for curious users and even for me it looks a bit easier to read than the full table in Notion. In the future we could provide some nice overview of all the events to the users, but I believe this is so far so good and it doesn't require too much extra maintenance .

Reply:
Maybe at least match versions somehow? We can bump version in package.json to `1.2` or smthing to avoid confusion

Reply:
I had an idea to avoid `SUITE_ANALYTICS_VERSION` and use a version from `package.json` but it is not possible to use only `major.minor`. There needs to be `major.minor.patch`. 

Is it good idea to put there version `0.1.18` into `@trezor/suite-analytics` `package.json` and trim the leading zero when reporting to analytics?